.rhcicd/do-nothing.yaml

```diff
@@ -1,25 +1,27 @@
 ---
-apiVersion: v1
+apiVersion: template.openshift.io/v1
 kind: Template
 metadata:
-  name: notifications-gw-do-nothing
+  name: notifications-do-nothing-gw
 objects:
-- apiVersion: batch/v1
-  kind: Job
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
   metadata:
-    name: notifications-do-nothing-job-${IMAGE_TAG}-${JOBID}
+    name: notifications-gw-do-nothing-job-${IMAGE_TAG}-${JOBID}
     annotations:
       ignore-check.kube-linter.io/unset-cpu-requirements : "Not required for this job that does nothing"  
-      ignore-check.kube-linter.io/unset-memory-requirements : "Not required for this job that does nothing"    
+      ignore-check.kube-linter.io/unset-memory-requirements : "Not required for this job that does nothing"  
   spec:
-    template:
-      spec:
-        containers:
-        - name: notifications-gw-do-nothing
-          image: quay.io/app-sre/ubi8-ubi-minimal:8.6-902
-          command: ["/bin/sh"]
-          args: ["-c", "exit 0"]
-        restartPolicy: Never
+    jobs:
+    - name: do-nothing-gw
+      restartPolicy: Never
+      schedule: "0 0 * * *"
+      podSpec:
+        image: quay.io/app-sre/ubi8-ubi-minimal:8.6-902
+        command: 
+          - /bin/sh
+          - -c 
+          - exit 0
 parameters:
 - name: IMAGE_TAG
   value: ''
```

Comment:
The `do-nothing.yaml` file from notifications-backend contains a `JOBID` param with some kind of regex. Is it needed here?

Reply:
https://github.com/RedHatInsights/notifications-backend/blob/ffccfe8968a1fb4803783ea5bebcb2fee20591dd/.rhcicd/do-nothing.yaml#L29-L31

Reply:
true, i separated the names, so we don't need jobid anymore in either of them